test/e2e/components/vl-http-error-message.js

```diff
@@ -47,11 +49,11 @@ class VlHttpErrorMessage extends VlElement {
     }
 
     async _getNormalImage() {
-        return this.shadowRoot.findElement(By.css('#image-normal'));
+        return new VlImage(this.driver, await this.shadowRoot.findElement(By.css('#image-normal')));
```

Comment:
er is geen src dependency van http-error-message naar vl-image, dus ofwel

- wordt vl-image niet gebruikt in http-error-message en mag je ook hier in het component-object niet koppelen
- moet er een src dependency komen van http-error-message naar image

Reply:
VlImage require 'optioneel' maken of een src dependency toevoegen naar vl-image blijft nog opgelost

Reply:
Hier wordt inderdaad geen vl-image gebruikt. @wautergu best `VlElement` returnen hier.